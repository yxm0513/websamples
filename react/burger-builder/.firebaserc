{
  "projects": {
    "default": "react-my-burger-f0ff8"
  }
}
