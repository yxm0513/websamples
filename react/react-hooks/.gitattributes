# GitHub linguist library

* linguist-vendored
*.js linguist-vendored=false
