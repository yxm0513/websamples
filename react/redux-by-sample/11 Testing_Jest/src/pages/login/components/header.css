.header {
  background-color: #5cb85c!important;
  color: white!important;
  padding: 20px;
}
