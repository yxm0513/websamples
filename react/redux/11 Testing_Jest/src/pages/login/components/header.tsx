import * as React from "react";
const styles = require('./header.css');

export const Header = () => {
	return (
		<div className={`panel-heading ${styles.header}`}>
			<h3 className="panel-title">Please sign in</h3>
		</div>
	);
}
