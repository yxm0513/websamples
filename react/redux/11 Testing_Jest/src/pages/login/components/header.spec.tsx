import * as React from 'react';
import { shallow } from 'enzyme';
import { Header } from './header';

describe('Header', () => {
  it('should render as expected', () => {
    // Arrange

    // Act
    const component = shallow(
      <Header />,
    );

    // Assert
    expect(component).toMatchSnapshot();
  });
});
