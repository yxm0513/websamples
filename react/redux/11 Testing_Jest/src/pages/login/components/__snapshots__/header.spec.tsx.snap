// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render as expected 1`] = `
<div
  className="panel-heading header"
>
  <h3
    className="panel-title"
  >
    Please sign in
  </h3>
</div>
`;
